LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015 Kyper Data Technologies
```

Comment:
Was this meant to be in the recipe?

Reply:
I just let the copyright of the ones who initially started the python highchart project. I think it is normal to keep it

Reply:
For clarity this was put at the top-level of the `staged-recipes` repo as opposed to in `recipes/vertica_highcharts`. Was this just an oversight?

Reply:
it is just a fork of another project but modified to fit our purposes. Why is it a problem to have such header?

Reply:
It's not a problem to have a license file in a recipe

It is a problem to add it to the top-level of `staged-recipes` as opposed to contained with a recipe. It makes it look like this license applies to `staged-recipes`, which it does not

Screenshot included below for clarity

<img width="1840" alt="Screenshot 2023-12-05 at 5 08 49 PM" src="https://github.com/conda-forge/staged-recipes/assets/3019665/7a32a98f-99f1-4b63-b5e6-82f2cfdab4fa">

Reply:
@afard this is what I told to Siting. I think we should add the copyrights to only one file and not write it everywhere (here it is for vertica highcharts, same applies to verticapy).

Reply:
@jakirkham, thanks for your feedback. You may notice that this PR is already merged, and the LICENSE file doesn't appear in the created feedstock: https://github.com/conda-forge/vertica_highcharts-feedstock/tree/main

Do you suggest any actions at this point?

Reply:
Yep the feedstock is only created with the files in the recipe directory

Would recommend

1. Fork the feedstock
2. Create a new branch
3. Commit the license file
4. Bump the `build/number`
5. Push all these changes to the fork
6. Open a PR to the feedstock

Happy to review if you need another pair of eyes

Reply:
Thanks @jakirkham for your advice.
Here is the PR. It would be kind of you to review it.
https://github.com/conda-forge/vertica_highcharts-feedstock/pull/3